ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,97 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def get_drpc_name(namespace):
+    """
+    Get the DRPC Resource Name
+    Args:
+        namespace (str): Name of namespace
+    Returns:
+        str: DRPC resource name
+    """
+
+    drpc_obj = ocp.OCP(kind=constants.DRPC, namespace=namespace,).get()[
+        "items"
+    ][0]
+    return drpc_obj["metadata"]["name"]
+
+
+def get_drpolicy_name():
+    """
+    Get DRPolicy Name
+    Returns:
+        str: DRPolicy name
+    """
+
+    drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY).get()["items"][0]
+    return drpolicy_obj["metadata"]["name"]
+
+
+def get_primary_cluster_name(namespace):
+    """
+    Get Primary Cluster Name based on Namespace
+    Args:
+        namespace (str): Name of the Namespace
+    Returns:
+        str: Primary Cluster Name
+    """
+    config.switch_acm_ctx()
+    drpc_resource_name = get_drpc_name(namespace=namespace)
+    drpc_obj = ocp.OCP(
+        kind=constants.DRPC,
+        namespace=namespace,
+        resource_name=drpc_resource_name,
+    ).get()
+
+    if drpc_obj.get("spec").get("action") == constants.ACTION_FAILOVER:
+
+        cluster_name = drpc_obj["spec"]["failoverCluster"]
+
+    elif drpc_obj.get("spec").get("action") == constants.ACTION_RELOCATE:
+        cluster_name = drpc_obj["spec"]["preferredCluster"]
+
+    else:
+        cluster_name = drpc_obj["spec"]["preferredCluster"]
+
+    return cluster_name
+
+
+def set_primary_cluster_context(namespace):
+    """
+    Set Primary Cluster Context based on Namespace
+    Args:
+        namespace (str): Name of the Namespace
+    """
+    cluster_name = get_primary_cluster_name(namespace)
+    config.switch_to_cluster_by_name(cluster_name)
+
+
+def get_secondary_cluster_name(namespace):
+    """
+    Get Secondary Cluster Name based on Namespace
+    Args:
+        namespace (str): Name of the Namespace
+    Returns:
+        str: Secondary cluster name
+    """
+    config.switch_acm_ctx()
+    drpc_resource_name = get_drpolicy_name()
```

Comment:
`drpc_resource_name` can be changed to `drpolicy_name`.

Reply:
Done